field.go

```diff
@@ -1,6 +0,0 @@
-package main
```

Comment:
i would actually prefer keeping this abstraction; in the future if we need to do things like enforce type safety on a particular field before sending it to segment (which i almost implemented in the first iteration), or enforcing a field value size cap, we could do it here. removing this class would make pushing any of those features backwards incompatible with old JSON schemas, which clients may not be happy about.